contributing.md

```diff
@@ -0,0 +1,112 @@
+# Contributing to Neighbor Army
+
+Thank you for contributing. We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+-   Reporting a bug
+-   Discussing the current state of the code
+-   Submitting a fix
+-   Proposing new features
+-   Becoming a maintainer
+
+## We Develop with Github
+
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
```

Comment:
Can we change it to from `develop` branch as that is our base branch?

Reply:
Good catch.  I just changed it